source/agora/node/Registry.d

```diff
@@ -359,14 +370,24 @@ public class NameRegistry: NameRegistryAPI
 
     ***************************************************************************/
 
-    private void doAXFR (in ZoneData zone, ref Message reply) @safe
+    private void doAXFR (string TemplateRegistry = "") (ZoneData!(TemplateRegistry) zone, ref Message reply) @safe
```

Comment:
I see you removed `in`. I suppose it's because the methods of `ZoneData` are not `const` ?